public/app/features/manage-dashboards/components/ImportDashboardForm.tsx

```diff
@@ -124,7 +124,7 @@ export const ImportDashboardForm: FC<Props> = ({
                     noDefault={true}
                     placeholder={input.info}
                     pluginId={input.pluginId}
-                    current={current[index]?.name}
+                    current={current[index]?.uid}
```

Comment:
Will this work for exports from older Grafana versions?

Reply:
It seems to work, I tested importing a dashboard from 7.5.5 with Prometheus Datasource(no `datasource.uid `there) and it worked, I think the backend is taking into account when we don't provision a datasource.uid